src/components/dataDisplays/IndividualsDisplay.jsx

```diff
@@ -25,51 +26,64 @@ export default function IndividualsDisplay({
         id: 'NAME',
       }),
       options: {
-        customBodyRender: (name, individual) => {
-          const id = get(individual, 'id', '');
+        customBodyRender: (_, individual) =>
+        {
+          const name = deriveIndividualName(
+            individual,
+            'FirstName',
+            'Unnamed Individual',
+          );
           return (
-            <Link href={`/individuals/${id}`}>
+            <Link href={`/individuals/${individual?.guid}`}>
               <Text variant="body2">{name}</Text>
             </Link>
           );
         },
       },
     },
     {
-      name: 'alias',
-      label: intl.formatMessage({
-        id: 'ALIAS',
-      }),
-      align: 'left',
-      options: { cellRenderer: cellRendererTypes.capitalizedString },
-    },
-    {
-      name: 'last_sighting',
-      label: intl.formatMessage({ id: 'LAST_SEEN' }),
-      align: 'left',
-      options: {
-        cellRenderer: cellRendererTypes.date,
-        cellRendererProps: { accessor: 'lastSeen' },
-      },
-    },
-    {
-      name: 'taxonomy',
-      label: intl.formatMessage({ id: 'SPECIES' }),
-      align: 'left',
-      options: { cellRenderer: cellRendererTypes.capitalizedString },
-    },
-    {
-      name: 'encounters',
-      label: intl.formatMessage({ id: 'SIGHTINGS' }),
+      name: 'adoptionName',
+      labelId: 'ADOPTION_NAME',
       align: 'left',
       options: {
-        customBodyRender: encounters => (
-          <Text variant="body2">
-            {get(encounters, 'length', '-')}
-          </Text>
-        ),
+        customBodyRender: (_, individual) =>
+        {
+          const adoptionName = deriveIndividualName(
+            individual,
+            'AdoptionName',
+            '-',
+          );
+          return <Text variant="body2">{adoptionName}</Text>;
+        },
       },
     },
+    // {
```

Comment:
commented out a bunch of features in the individuals table that we dont currently have the response for in elasticsearch. these will hopefully reappearing soon